powerline/bindings/lemonbar/powerline-lemonbar-multimonitor.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+# vim:fileencoding=utf-8:noet
+from __future__ import (unicode_literals, division, absolute_import, print_function)
+import sys
+import time
+import re
+import subprocess
+
+from threading import Lock, Timer
+from argparse import ArgumentParser, REMAINDER
+
+from powerline.lemonbar import LemonbarPowerline
+from powerline.lib.monotonic import monotonic
+from powerline.lib.dict import mergedicts
+
+
+if __name__ == '__main__':
+	parser = ArgumentParser(
+		description='Powerline BAR bindings.',
+		usage='%(prog)s [-h] [--i3] [--height HEIGHT] [--theme THEME [THEME ...]] -- args')
+	parser.add_argument(
+		'--i3', action='store_true',
+		help='Subscribe for i3 events.'
+	)
+	parser.add_argument(
+		'--height', default='',
+		help='Bar height'
+	)
+	parser.add_argument(
+		'--theme', '-t',
+		default=[], nargs='+',
+		help='makes all outputs that containing a string use the specified theme. Format like "out=theme"'
+	)
+	parser.add_argument(
+		'args', nargs=REMAINDER,
+		help='Extra arguments for lemonbar'
+	)
+	args = parser.parse_args()
+	theme_rules = {}
+	for theme in args.theme:
+		out, theme = theme.split('=', 1)
+		theme_rules[out] = theme
+
+	active_screens = [match.groupdict() for match in re.finditer(
```

Comment:
Yeah, it doesn't react to changes in configuration. That would mean more dependencies and probably be even less universal than `xrandr` already is.

Also usually you have to reconfigure the monitors with some sort of script anyway (like xrandr) when you plug/unplug them and often you need to do more things like reapplying the wallpaper etc. When that happens you can simply restart the script and it will autodetect again.